tests/release_tests/test_launch/README.md

```diff
@@ -3,8 +3,13 @@
 These tests are intended to be run before any release of Launch, to ensure common use cases haven't regressed. The tests shouldn't mock anything related to the wandb server, and should make actual API calls the same way a user would.
 
 Prerequisites:
+
 - You must be logged into a Weights & Biases account that has access to the entity `launch-release-testing`
 - You will also need to have AWS credentials - `~/.aws/credentials` should have your creds set
 - If you are testing local changes, be sure to build an image using `python ./tools/build_launch_agent.py --tag wandb-launch-agent:latest`
 
 If you are a W&B employee and have any questions or want to request access to the `launch-release-testing` entity, contact Tim Hays in Slack
+
+## Vertex regression tests
+
+These tests depend on google cloud resources and wandb resources. If you are interested in running these tests locally, please contact Ben Sherman in slack.
```

Comment:
We should probably have something here, even if it's as vague as "ensure you have access to X and are logged into Y", for the sake of people who have all the right permissions but need a refresher on how to set up their environment (e.g. yourself in six months 🙂)